runtime/src/stakes.rs

```diff
@@ -15,9 +15,8 @@ pub struct Stakes {
     /// stake_delegations
     stake_delegations: HashMap<Pubkey, Delegation>,
 
-    /// unclaimed points.
-    //  a point is a credit multiplied by the stake
-    points: u64,
+    /// unused
+    unused: u64,
```

Comment:
@rwalker-com Well, I just noticed that this changes ABI digest when merged. This field renaming should be no harm. Could you rebase this pr to be after #8012 and update the ABI digest of `Bank`?

Reply:
I've rebased, but how do I ensure I've done the right thing here with a local test?

Reply:
I don't think we have a way to add a test here, updating the digest is just a manual human thing

Reply:
"What steps does one use test the abi stuff locally?" was really my question.

Reply:
Just the 🧠, I'm afraid.  There's nothing nice here :-/

Reply:
sorry, even more specific: how do I run a test locally that fails when checking this hash?  I have tried a fair few things and failed to reproduce the failure that shows up in CI.

Reply:
Oh, sorry.  I see what you mean.   You just need to use a recent nightly I think, is that right @ryoqun?

Reply:
Yeah, just plain official recent nightly should be enough.

Reply:
please guys: I'm a good ways out of the normal dev flow these days, using an old personal Mac or a WSL box to dev and test.  `cargo test` never completes locally for me.  I was looking for STR locally for this specific failure in CI.

perhaps it's:

`cargo +nightly test --package solana-runtime abi -- --nocapture` ?

Reply:
@rwalker-com Yeah exactly that will work.

I use this particular arguments (`--lib` reduces the build time a bit and abi tests doesn't need them): 

```
cargo test --lib -- --test-threads 2 test_abi_ --nocapture
```

Reply:
Oh, my rustup default is to use `nightly`, btw.

Reply:
ok.  once one has reproduced this test failure, there's this suggestion:
`Confirm the diff by rerunning before and after this test failed with SOLANA_ABI_DUMP_DIR!', runtime/src/serde_snapshot/tests.rs:285:5`